api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
```

Comment:
First of all, I have added the compat data despite the ongoing discussion in #190 about representing the "secure context" requirement.

In my opinion, a browser could be considered compatible if it offers the feature in the secure context as defined by the spec. On the other hand, I wouldn't consider a browser incompatible, if it also offers the feature outside of the [required] secure context, but instead point this out in a `"notes"` entry.